.github/workflows/clang-tidy-rocm.yaml

```diff
@@ -0,0 +1,53 @@
+name: Linter-ROCM
+
+on: [push, pull_request]
+
+env:
+  OPEN_UCX_LINK: https://github.com/openucx/ucx
+  OPEN_UCX_BRANCH: master
+  CLANG_VER: 12
+  ROCM_VER: 5-4
+  LIBRARY_PATH: /tmp/ucx/install/lib
+  LD_LIBRARY_PATH: /tmp/ucx/install/lib
+jobs:
+  clang-tidy:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y --no-install-recommends wget lsb-core software-properties-common gpg curl
+        sudo apt-get install -y --no-install-recommends libibverbs-dev rdma-core rdmacm-utils libibumad-dev
```

Comment:
minor, these packages are needed for TL SHARP, can be skipped here

Reply:
thank you, I removed the rdma and infiniband packes from the install list.